tools/isobuild/import-scanner.js

```diff
@@ -249,19 +249,6 @@ export default class ImportScanner {
       // something plausible. #6411 #6383
       const absPath = pathJoin(this.sourceRoot, file.sourcePath);
 
-      const dotExt = "." + file.type;
-      const dataString = file.data.toString("utf8");
```

Comment:
Accessing `file.data` here (in `addInputFiles`) would have forced every file to be fully compiled, regardless of whether it was used.